perf/load/common.sh

```diff
@@ -68,3 +73,34 @@ function start_servicegraphs() {
     sleep 30
   }
 }
+
+# This function assumes preconfigured ASM multi clusters ready.
+# Here is a simple setup guide for example: https://cloud.google.com/service-mesh/docs/gke-install-multi-cluster
```

Comment:
This is some `ASM` specific code here.
Can we move it into vendors/google too?

Reply:
done